### ISSUE TITLE
No updates found for the laravel/homestead box

### ISSUE THREAD

**Issue by andrey-helldar** [laravel/homestead#924]
For a long time I installed the `laravel/homestead` box of `5.x` version.
I updated it with the command `vagrant box update`.
Now version `6.1.0` is installed, but on site [vagrantup](https://app.vagrantup.com/laravel/boxes/homestead) there is a newer version - `6.3.0`.

When I installed my version, it was downloaded for `virtualbox` provider.
I recently installed a fresh version on another computer and when I installed, I downloaded a box for the `vmware` provider (given that the settings specified `virtualbox` provider.).

Executing command `vagrant box update` causes an error that the box with that name already exists.

Tell me, how can I update the version of the box without serious consequences?

```yaml
---
ip: "192.168.10.10"
memory: 2048
cpus: 1
provider: virtualbox

# ...other settings...
```

**Comment by svpernova09** [comment]
The 7.3.0 version has not yet been released. Extensive conference travel + some personal stresses have delayed the release. My plan is to get it released this weekend.

**Comment by andrey-helldar** [comment]
Interesting. Before I wrote the article, I checked the page and clearly saw the version 6.3.0 on it (though the inscription `currently released version` was in version 6.2.0, as it is now).
Nevertheless, on my computer is 6.1.0 for the virtual box.

On another computer with `virtualbox` installed, the image downloaded for `vmware` works. This is normal?

**Comment by svpernova09** [comment]
> Interesting. Before I wrote the article, I checked the page and clearly saw the version 6.3.0 on it (though the inscription currently released version was in version 6.2.0, as it is now).
Nevertheless, on my computer is 6.1.0 for the virtual box.

Since I'm not entirely sure when I'm going to release 6.3.0 I've removed it so it won't show up during vagrant commands.

> On another computer with virtualbox installed, the image downloaded for vmware works. This is normal?

The current version of Virtualbox is [box v6.1.0](https://app.vagrantup.com/laravel/boxes/homestead/versions/6.1.0) The current version of VMware is [box v6.2.0](https://app.vagrantup.com/laravel/boxes/homestead/versions/6.2.0). 

Normally we keep all providers at the same version however with 6.2.0 the only change was specific to the VMware provider so we only built that provider for the release.

**Comment by andrey-helldar** [comment]
> The current version of Virtualbox is box v6.1.0 The current version of VMware is box v6.2.0.
> 
> Normally we keep all providers at the same version however with 6.2.0 the only change was specific to the VMware provider so we only built that provider for the release.

Interestingly, I tried to install the box for virtualbox, but it was installed for vmware.
The configuration looks like this:
```yaml
---
ip: "192.168.10.10"
memory: 2048
cpus: 1
provider: virtualbox

# ...other settings...
```
and everything works without problems.

Installed via command `vagrant box add laravel/homestead` after setting `Homestead.yaml` file.
